packages/expo-notifications/README.md

```diff
@@ -723,6 +724,38 @@ Cancels all scheduled notifications.
 
 A `Promise` resolving once all the scheduled notifications are successfully cancelled or if there are no scheduled notifications.
 
+### `getNextTriggerDateAsync(trigger: SchedulableNotificationTriggerInput): Promise<number | null>`
+
+Allows you to check what will be the next trigger date for given notification trigger input.
+
+#### Arguments
+
+The schedulable notification trigger you would like to check next trigger date for (of type [`SchedulableNotificationTriggerInput`](#schedulablenotificationtriggerinput)).
+
+#### Returns
+
+If the return value is `null`, the notification won't be triggered. Otherwise, the return value is the Unix timestamp in millseconds at which the notification will be triggered.
```

Comment:
```suggestion
If the return value is `null`, the notification won't be triggered. Otherwise, the return value is the Unix timestamp in milliseconds at which the notification will be triggered.
```